src/index.js

```diff
@@ -1,8 +1,13 @@
 import {defaultSelectorHandlers} from './generate';
 import makeExports from './exports';
+import * as types from './types';
 
 const useImportant = true; // Add !important to all style definitions
-export default makeExports(
+const exports = makeExports(
     useImportant,
     defaultSelectorHandlers
 );
+
+exports.types = types;
```

Comment:
What do you think about adding these types to the no-important export as well? That way, we don't have to important from both libraries?

Reply:
When the entire f4 app is built, could that lead to this code being built in twice since it uses both libraries? Is there a case when we use just one of these libraries elsewhere?

Reply:
Actually, are both versions installed anyway? I just looked at package.json in react-components, and we only install @factoryfour/aphrodite but use the no-important build as well. In that case, then I guess it's fine to just define it in one place. I just though that it looked a little weird to important from both builds in a single file, but this would be an unnecessary duplication indeed.

Reply:
You're right, both versions are always installed! In that case, definitely more concerned about code duplication

Reply:
Yep, then I agree 100% with only exporting this here! I think you just have to resolve the Travis tests and this is good to go!

Reply:
hm this is weird - the tests are failing for me on master?

Reply:
Huh, did we setup Travis after making the last release?